spec.bs

```diff
@@ -497,6 +506,22 @@ The specification adds the following steps to the [=HTTP fetch=] algorithm, befo
 1. If |redeem response result| is a [=network error=], return |issue response result|.
 
 
+Integration with XMLHttpRequest {#xhr}
+====================================
+
+An XMLHttpRequest has an associated {{PrivateToken}} object specifying the {{OperationType}} to execute against the request.
```

Comment:
You need to use this field somewhere to inform the actual request. Is that a monkeypatch to https://xhr.spec.whatwg.org/#the-send()-method to fill in a new Request field?

Reply:
Added details for send() monkeypatch